frontend/views/components/AvatarUser.vue

```diff
@@ -38,7 +38,7 @@ export default {
       console.debug(`Looking for ${this.username} profile picture`)
       const userContractId = await sbp('namespace/lookup', this.username)
       if (!userContractId) {
-        console.error(`AvatarUser: ${this.username} doesn't exist!`)
+        console.warn(`AvatarUser: ${this.username} doesn't exist!`)
```

Comment:
Why the change?

Reply:
Because I think this is not an error... Nothing wrong happened with the app... This can happen if we try to look for a user that doesn't exist.
Example1: looking for a member name that was invited but it didn't join the group yet.) Example2: looking for a username that doesn't exist anymore (deleted the account)